src/pyscipopt/scip.pxd

```diff
@@ -1637,6 +1670,30 @@ cdef class Solution:
     @staticmethod
     cdef create(SCIP* scip, SCIP_SOL* scip_sol)
 
+cdef class DomainChanges:
+    """Set of domain changes."""
+    cdef SCIP_DOMCHG* scip_domchg
+
+    @staticmethod
+    cdef create(SCIP_DOMCHG* scip_domchg)
+
+cdef class BoundChange:
+    """Bound change."""
+    cdef SCIP_BOUNDCHG* scip_boundchg
+
+    @staticmethod
+    cdef create(SCIP_BOUNDCHG* scip_boundchg)
+
+cdef class Branching:
+    """Branching decision."""
+    cdef SCIP_VAR* scip_var
+    cdef SCIP_Real scip_bound
+    cdef SCIP_BOUNDTYPE scip_boundtype
+
+    @staticmethod
+    cdef create(SCIP_VAR* scip_var, SCIP_Real scip_bound,
+                SCIP_BOUNDTYPE scip_boundtype)
+
```

Comment:
So here I am not sure if we really want to have this in the interface. This is a convenience class to store the triples (var, bound, boundtype). I would say no, we probably don't want to maintain this nor write documentation for it, but @mattmilten, @mueldgog  what do you think?

Reply:
I've removed this class.